packages/ui-react/src/components/Subscription/Subscription.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import useBillingPortal from '@jwp/ott-hooks-react/src/useBillingPortal';
+import type { Subscription as SubscriptionType } from '@jwp/ott-common/types/subscription';
+import { formatLocalizedDate, formatPrice } from '@jwp/ott-common/src/utils/formatting';
+import { useConfigStore } from '@jwp/ott-common/src/stores/ConfigStore';
+
+import userStyles from '../../pages/User/User.module.scss';
+import Button from '../Button/Button';
+import LoadingOverlay from '../LoadingOverlay/LoadingOverlay';
+
+import styles from './Subscription.module.scss';
+
+const ActiveSubscription: React.FC<{ subscription: SubscriptionType }> = ({ subscription }) => {
+  const { t, i18n } = useTranslation('user');
+  const isAccessBridgeEnabled = useConfigStore((state) => !!state.settings?.apiAccessBridgeUrl);
```

Comment:
Do we need "Subscription" component without access bridge?

Reply:
Basically the correlation is rather: if it's offers (only Cleeng ATM) then "Payment", if it's acces plans (SVOD, only JWP ATM) then "Subscription".

Reply:
To answer your question fully - no, the way things are right now, the Subscription component and what it's supposed to render relies on the access bridge.  It's the only way for the `Go to Stripe Billing portal` button to serve its purpose.

Reply:
Thanks. I was just thinking if we could add this condition on the top level to make it clear.

Reply:
The `Subscription` component is a route that gets redirected to from menu options that rendered conditionally in `User.tsx` and `UserMenuNav.tsx` anyway.